automation/jleechanorg_pr_automation/jleechanorg_pr_monitor.py

```
@@ -1332,6 +1391,38 @@ def main():
 
     monitor = JleechanorgPRMonitor()
 
+    if args.codex_update:
+        print("🤖 Running Codex automation (first 50 tasks)...")
```

Ruff (0.14.8)

`print` found

Remove `print`

(T201)